giraffe/src/utils/useHoverPointIndices.ts

```diff
@@ -118,6 +146,8 @@ const buildIndex = (
   const yBinCount = Math.ceil(height / INDEX_BIN_WIDTH) + 1
 
   const indexTable = {
+    maxCountPerXBin: Math.ceil(TOTAL_POINTS_PER_HOVER_DIMENSION / xBinCount),
+    maxCountPerYBin: Math.ceil(TOTAL_POINTS_PER_HOVER_DIMENSION / yBinCount),
```

Comment:
are `xBinCount` and `yBinCount` guaranteed to be nonzero? could `Math.ceil(width / INDEX_BIN_WIDTH)` ever evaluate to -1 (so that adding 1 sums to 0)?